README.md

```diff
@@ -32,21 +43,23 @@ The top-level folder of the repository is organized as follows. Different code e
 - COULOMB: Computation of the electric potential created by a set of charges in an n x n 2D plane.
 - HACCmk: Short force evaluation kernel of the HACC application.
 - MATMUL: Matrix multiplication.
+- MbedTLS: Open source TLS C library.
 - NPB CG: Conjugate Gradient, irregular memory access and communication-
 - PI: PI number approximation.
+- VMS: Virtual Molecule Screening pure_c.
 
-Inside each one of them, the `serial` subfolder corresponds to the original sequential (ie. non-parallelized) code. Additionally, one or more subfolders corresponding to performance-optimized versions are provided. The `pwa-omp-multi` folder corresponds to an OpenMP multi-threaded version generated by Parallelware Analyzer, which requires an OpenMP compiler like `gcc` or `clang` properly installed. The `pwa-acc-offload` folder corresponds to an OpenACC GPU offloading version generated by Parallelware Analyzer, which requires and OpenACC compiler like `nvc` properly installed. 
+Inside each one of  (except MbedTLS), the `serial` subfolder corresponds to the original sequential (ie. non-parallelized) code. Additionally, one or more subfolders corresponding to performance-optimized versions are provided. The `pwa-omp-multi` folder corresponds to an OpenMP multi-threaded version generated by Codee, which requires an OpenMP compiler like `gcc` or `clang` properly installed. The `pwa-acc-offload` folder corresponds to an OpenACC GPU offloading version generated by Codee, which requires and OpenACC compiler like `nvc` properly installed. 
```

Comment:
If I remember correctly @manuelarenaz  requested that all the tests in the performance-demos follow the same format. The user would then just run `make` in the root directory to measure before and after time.